ui.R

```diff
@@ -2,99 +2,124 @@ library(shiny)
-        tags$head(
-          tags$style(HTML("#go{background-color: #367fa9}"))
-        ),
+        tabName = "Overview",
+        withMathJax(),
+        h1(strong("Hypothesis Test for Means")),
+        p("In this app the goal is to learn about the reasoning of a hypothesis test about proportions."),
+        h2("Instructions"),
+        tags$li("In Part 1 you will look at how the population distribution of all the players' free throw percentages is affected by filtering (restricting attention to a subpopulation)."),
         div(
-          style = "text-align: center;",
-          bsButton("go1", "Go to filtering exploration", icon = icon("bolt"))
-        ),
-        h4(tags$li("In Part 2 you will explore hypothesis tests about an individual players' free throw percentages.")),
-        tags$head(
-          tags$style(HTML("#go{background-color: #367fa9}"))
+          style = "text-align: center",
+          bsButton(
+            input = "go1",
+            label = "Explore", 
+            size = "large",
+            icon = icon("bolt")
+          )
         ),
+        tags$li("In Part 2 you will explore hypothesis tests about an individual players' free throw percentages."),
         div(
           style = "text-align: center;",
-          bsButton("go2", "Go to hypothesis tester", icon = icon("bolt"))
+          bsButton(
+            input = "go2",
+            label = "Challenge", 
+            size = "large",
+            icon = icon("bolt")
+          )
         ),
         br(),
-        h3(strong("Acknowledgements:")),
-        h4("This app was developed and programmed in 2017 by David Robinson. The hypothesis testing features in part 2 were edited and improved with additional programming in 2018 by Ryan Voyack."),
-
         br(),
+        h2("Acknowledgements"),
+        p("This app was developed and programmed in 2017 by David Robinson. The hypothesis testing features in part 2 were edited and improved with additional programming in 2018 by Ryan Voyack. The updated version of this app was improved in 2020 by Xuefei Wang."),
+        br(),
+        img(src = "fthrow2.png", height = 250, width = 650, algin = "middle"),
         br(),
-        img(src = "fthrow2.png", height = 250, width = 650, algin = "middle")
+        br(),
+        br(),
+        div(class = "updated", "Last Update: 5/13/2020 by XW.")
       ),
-
-      # Define the content contained within part 1 ie. tabname "filtering"
+      
+      # This is content for the second tab, "Explore"
       tabItem(
-        tabName = "filtering",
-        div(
-          style = "display: inline-block;vertical-align:top;",
-          tags$a(href = "https://shinyapps.science.psu.edu/", tags$img(src = "homebut.PNG", width = 15))
-        ),
+        tabName = "Explore",
+        withMathJax(),
         fluidRow(
```

Comment:
Let's add some type of h2 text at the top of the page to give a title to this tab.